### PR TITLE
Updated list of companies: Esper, Mr. Number, Skydeck

### PR DESCRIPTION
**Pull request by mjambon**
Added Esper to list of companies, removed mention of Skydeck (defunct), updated Mr. Number.

### CODE REVIEW

File: site/learn/companies.md [hunk 1]
```diff
@@ -129,6 +129,28 @@
 </dl>
 <dl class="row">
     <dt class="span3">
+        <img src="/img/default.png" alt="Esper">
+    </dt>
+    <dd class="span5">
+        <h2><a href="http://esper.com/">Esper</a>, United States</h2>
+        <p>Esper's software assists
+        assistants. In doing so, we save
+        the time of executives. We want to lead the way to a more
```
Comment (Chris00) [inline site/learn/companies.md:138]: > assists assistants

Hum, please reword.

**Comment by Chris00** [comment]
I merged it but you may want to improve some wording.

File: site/learn/companies.md [hunk 2]
```diff
@@ -129,6 +129,28 @@
...
+        productive world with an assistant for every professional and
```
Comment (Chris00) [inline site/learn/companies.md:139]: > assistant for every professional and Esper for every assistant.

On first read I was not sure what you meant.  Not sure "an assistant for every professional" is really seen an an improvement...

Comment (mjambon) [inline site/learn/companies.md:139]: Read "for every professional" not "per professional", of course. Not sure how to rephrase yet.

**Pull request merged**